spec/intl.html

```diff
@@ -1338,6 +1338,101 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
             1. Return _result_.
           </emu-alg>
         </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatefromfields">
+          <h1>CalendarDateToISO ( _calendar_, _fields_, _overflow_ )</h1>
+          <p>The abstract operation CalendarDateToISO performs implementation-defined processing to convert _fields_, which is a Record that describes a date in the calendar represented by the string _calendar_ to the ISO calendar and returns the corresponding day, month and year values as a Record.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateaddition">
+          <h1>CalendarDateAddition ( _calendar_, _date_, _duration_, _overflow_ )</h1>
+          <p>The abstract operation CalendarDateAddition performs implementation-defined processing to add _duration_ to _date_ in the context of the calendar represented by the string _calendar_ and returns the corresponding day, month and year of the result in the ISO calendar values as a Record.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedifference">
+          <h1>CalendarDateDifference ( _calendar_, _one_, _two_, _largestUnit_ )</h1>
+          <p>The abstract operation CalendarDateDifference performs implementation-defined processing to find the difference between the two dates _one_ and _two_ in the context of the calendar represented by the string _calendar_ and returns the corresponding years, months, weeks and days as a Record.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateera">
+          <h1>CalendarDateEra ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateEra performs implementation-defined processing to find the era for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateerayear">
+          <h1>CalendarDateEraYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateEraYear performs implementation-defined processing to find the year of the current era for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateyear">
+          <h1>CalendarDateYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateYear performs implementation-defined processing to find the year for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
```

Comment:
Should document that this is a 1-based signed number relative to a calendar-specific "epoch year", which is typically the first year of a particular era for calendars that use eras, or is year 1 or 0 for calendars that don't use eras. 

For example, the `'gregory'` calendar's epoch year is 1 CE, so `year` for 3CE should be `3`, and `year` for 2 BCE should be `-1`.

Also, for the ICU calendars we should probably specify which era is used as the epoch era and what the epoch date is in both calendar and ISO dates. It would be bad if different implementations picked different epoch dates!

Reply:
Same as https://github.com/tc39/proposal-temporal/pull/1928#discussion_r755043313, I agree that the calendar-specific behavior should be specified, just not here.